CHANGES

```diff
@@ -4,6 +4,8 @@ master
 * Jenkins support:
   - Fail Jenkins checks when job is unknown
   - Use [jenkinsapi](https://pypi.python.org/pypi/jenkinsapi) to talk to Jenkins
+  - Add option to specify multiple Jenkins backends
+  > NOTE: This update will delete any recent status check results for jenkins checks
```

Comment:
no i don't think so, it's all internal for now.

Maybe once we've done some more refactoring and pulled them all out into plugins